Added neutral reinforcements

nymanjens opened the pull request:
This is my proposal for stuff to put on the turn track so we don't forget to advance it. I think it can be a fun little advantage for the losing team.

Tantali commented:
I wouldn't mind having some random or player-chosen event somewhere after the move-token-phase, but this particular suggestion I'm not a fan of. The losing party having a broad choice of placing a neutral infantry feels like make-shift addition to the rules. It doesn't tie in well I think.

317070 commented:
How do you 'decide as a team'? What if one player wants A, and another wants B?

And a battle with one attacker can never be won by the attacker, right? So I don't really see the point there either.

nymanjens commented:
> How do you 'decide as a team'? What if one player wants A, and another wants B?

That is an unfair question because you could ask the same question about where neutral units should retreat when they retreat.

> And a battle with one attacker can never be won by the attacker, right? So I don't really see the point there either.

1/3 chance of killing a unit of your choice

317070 commented:
> That is an unfair question

Whoa. Don't take it personally. :confused: I am only trying to help.

> because you could ask the same question about where neutral units should retreat when they retreat.

That is a good point. I forgot neutral units could retreat.

I would maybe change it slightly. Maybe those units can be put on the score track of the teams? That way, if a team gains 2 points, 2 neutral units get put in at the same time. (Although that defeats the point of forcing the players not to forget the move counter)

I agree with Robin that it feels kind of 'off'. Why the losing team? What when both teams are equal?

Edit: After going through this thread: https://boardgamegeek.com/thread/1091940/game-mechanic-question-best-way-reliably-keep-trac
A different proposal: what if you start with 4 command tokens, and every/some turns 1 more token becomes available for each player. Which one that is, is indicated on the turn counter track. It would make token trading more important (as there is more scarcity). It could be made into theme, by tying it to an invention at the time: tanks for a move token, tunnel warfare for a trench token, war bonds for an invest token, conscription for a recruit token.

As is the case in for instance terra mystica, the order of these could be selected randomly in the beginning of the game, after which you can calculate which strategy befits best based upon when you will receive which token. And as in terra mystica, then you simply don't forget to move turns.

Or alternatively: pure UI-design-wise, you could split every turn into 2 parts: a preparation part and a move part. They are both indicated in the turn counters. It is much harder to forget or to go by unnoticed, as for half of each turn, the pawn would clearly be in the wrong spot.

nymanjens commented:
> > That is an unfair question
> 
> Whoa. Don't take it personally. :confused: I am only trying to help.

I didn't take it personally, I just wanted to point out that this is probably a non-issue.

> Why the losing team?

The losing team to balance the game (by a tiny amount) 

> What when both teams are equal?

As stated in the rules, nothing happens

Nobody is really happy with this proposal and I was kind of unsure about it too. 

> A different proposal: what if you start with 4 command tokens, and every/some turns 1 more token becomes available for each player

There are things I like about it and things I don't:

* It's probably pretty effective at not forgetting to advance the turn counter
* This is not Terra Mystica. It's almost impossible to plan >1 turn ahead
* In general, I'm worried about adding stuff that impracts the game too much

Note that we removed token trading from `master` to discourage the guerrilla tactic
 
> Or alternatively: pure UI-design-wise, you could split every turn into 2 parts: a preparation part and a move part

We have now split the game into 7 parts and I don't see a very clear split in there. If the UI could bear it, it would maybe be better to split every turn into 7 parts as to help players with doing everything in the right order (this is how Blood Rage works, but they have only 3 turns and 4 parts).

That said, I think 2 parts and 7 parts will both be forgotten by some types of players.

**My conclusion**: Currently, I don't think my or any other proposal is a clear improvement and it may solve only a relatively small problem. Having an extra turn does not necessarily decrease the fun of the game as it is pretty arbitrary number anyway.
